src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -424,12 +427,68 @@ export default class MultipleCategoryBarPlot extends React.Component<
     }
 
     @computed get labels() {
+        console.log(this.data, 'thisisis');
+        interface TotalSumItem {
+            majorCategory: string;
+            sum: number;
+            minorCategory: {
+                name: string;
+                count: number;
+                percentage: number;
+            }[];
+        }
+        this.data.forEach(item => {
```

Comment:
for array functions such as `forEach`, lets try using the lodash version like _.forEach(this.data,` ...)` when we can throughout